Concierge/Expose-Bank/Connect-Sample/ViewController.swift

```diff
@@ -0,0 +1,57 @@
+//
+//  ViewController.swift
+//  Connect-Sample
+//
+//  Created by Caio on 2022-12-07.
+//
+
+import UIKit
+import FlybitsConcierge
+
+class ViewController: UIViewController {
+    @IBOutlet private var connectButton: UIButton!
+    @IBOutlet private var disconnectButton: UIButton!
+    @IBOutlet private var physicalDeviceIdentifierTextField: UITextField!
+    @IBOutlet private var customIdentifierTextField: UITextField!
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+
+        if let identifier = UIDevice.current.identifierForVendor?.uuidString {
+            self.physicalDeviceIdentifierTextField.text = identifier
+        } else {
+            print("Failed to get the `UIDevice.current.identifierForVendor?.uuidString` information")
+        }
+    }
+
+    override func viewWillAppear(_ animated: Bool) {
+        super.viewWillAppear(animated)
+
+        self.connectButton.isHidden = Concierge.isConnected
+        self.disconnectButton.isHidden = !Concierge.isConnected
+    }
+
+    @IBAction func connectOnTouchUpInside(sender: UIButton) {
+        let physicalIdentifier = self.physicalDeviceIdentifierTextField.text
```

Comment:
@caiodias Is there a way to save the values so you can use the check status function?

Reply:
Which values?
the connect and disconnect status?

Reply:
@caiodias we need some comments to stress the importance of using the same values on subsequent calls to connect